src/helm/proxy/clients/huggingface_client.py

```diff
@@ -236,9 +236,16 @@ def do_it():
                     ]:
```

Comment:
Explain why these models need a different logic...?  Does this need to be special cased?

Reply:
ok!